README.md

```diff
@@ -109,6 +109,7 @@ s1, err = c.CreateStation("<station-name>",
  memphis.SendPoisonMsgToDls(<bool>), // defaults to true
  memphis.SendSchemaFailedMsgToDls(<bool>), // defaults to true
  memphis.TieredStorageEnabled(<bool>) // defaults to false
+ memphis.PartitionsNumber(<int>) // default is 1 partition
```

Comment:
add a section on the produce and consumer that if a station has more than 1 partition it makes round robin